DSL/Liquibase/liquibase.properties

```diff
@@ -0,0 +1,6 @@
+changelogFile: /changelog/master.yml
+url: jdbc:postgresql://localhost:5433/classifier
+username: root
+password: root
```

Comment:
We shouldn't ideally reveal this in the codebase right @kalsaraM ?

Reply:
this password ovverrid byscript